scripts/close_old_issues.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import requests
+import json
+
+from datetime import datetime
+
+def parse_arguments():
+    """ parse command line arguments """
+    parser = argparse.ArgumentParser(description="Auto close issues")
+    parser.add_argument(
+        "-token",
+        required=True,
+        type=str,
+        help="Github access token",
+    )
+    return parser.parse_args()
+
+def get_issues(token: str):
+    params = {"Accept": "application/vnd.github+json",
+              "Authorization": f"token {token}",
+              "X-GitHub-Api-Version": "2022-11-28"}
+    url = f"https://api.github.com/repos/ewlu/gcc-precommit-ci/issues?per_page=100&state=open"
+    r = requests.get(url, params)
+    issues = json.loads(r.text)
+    filtered = [issue for issue in issues if 'pull_request' not in issue.keys()]
+    return filtered
+
+def get_patch_id(issue):
+    body = issue["body"].split("\n")
+    for line in body():
+        if line == "":
+            continue
+        if "Patch id" in line:
```

Comment:
Can you check for `"Patch id:"`? that way we're guarenteed to have a `:` to split on. Probably won't matter in practice but might be a nice guardrail.